jamshidbek/db.sql

```diff
@@ -0,0 +1,187 @@
+-- 1
+CREATE TABLE studentt (
+    student_id SERIAL PRIMARY KEY,
+    first_name VARCHAR(50),
+    last_name VARCHAR(50),
+    date_of_birth DATE,
+    gpa DECIMAL(3, 2)
```

Comment:
you should not have GPA instead you need enrollment_year or something that can save info of enrollment year